CalendarFXView/src/main/java/com/calendarfx/model/EditOperation.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2017 Dirk Lemmermann Software & Consulting (dlsc.com)
+ * <p>
+ * This file is part of CalendarFX.
+ */
+package com.calendarfx.model;
+
+/**
+ * possible edit operations with an entry
```

Comment:
Start javadocs with UPPER case.

Also include link to the method where this enum is used.